### DISCUSSION TITLE
Filter by country

### DISCUSSION THREAD

**Discussion by henrikzabel** [komoot/photon#677] [Q&A]
https://photon.komoot.io/api?q=Berlin&limit=3&osm_tag=country:germany

I don't know a way to filter by country. My example does not work obviously ;/

Maybe someone can help me? :)

**Comment by lonvia** [comment]
There is no function for filtering by country. You can only add the country to the search term (which isn't the same but hopefully still improves the result).

**Comment by uqlwil35** [comment]
I implemented this by first querying for the country itself to gets its coordinates, then biassing a second search towards the country's coordinates and finally filtering out non-matching results.
